.github/workflows/release.yaml

```diff
@@ -0,0 +1,30 @@
+# Run this workflow only on merged release-please PRs
+name: Release PR Merged
+on:
+  push:
+    branches: ["main"]
+    # look for the specific files that are changed in a release-pr
+    paths:
+      - CHANGELOG.md
+      - across_server/__init__.py
+      - .release-please-manifest.json
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  id-token: write
+  deployments: write
+  pull-requests: write
+  contents: read
+
+jobs:
+  release-please:
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: googleapis/release-please-action@v4
+        with:
+          token: ${{ secrets.RELEASE_PLEASE_TOKEN }}
```

Comment:
did you create the PAT?
https://github.com/googleapis/release-please-action?tab=readme-ov-file#basic-configuration